packages/shared/src/utils/cfd/cfd.js

```diff
@@ -111,7 +111,11 @@ export const getCFDAccountDisplay = ({
         else cfd_account_key = 'cfd';
     }
 
-    return CFD_text_translated[cfd_account_key]();
+    const cfd_account_display = CFD_text_translated[cfd_account_key]();
+
+    if (market_type === 'synthetic' && platform === CFD_PLATFORMS.DXTRADE) return 'Synthetic';
```

Comment:
It is not related to this card but I think it's better to add a TODO for this condition because when this card gets merged, there is no `market_type === 'synthetic'` in the `DXTRADE` platform and this condition is useless unless we change it to something like this:

 ```suggestion
    if (market_type === 'all' && platform === CFD_PLATFORMS.DXTRADE) return 'Deriv X';
```